hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -193,10 +196,11 @@ public synchronized void initialize() throws IOException {
       if (token != null) {
         UserGroupInformation.getCurrentUser().addToken(token);
```

Comment:
Should we just pass down the token from the caller directly and bypass the UGI addToken/getTokens completely?